src/views/GovernanceReviewTeam/GRBReview/index.tsx

```diff
@@ -153,6 +172,68 @@ const GRBReview = ({ id, state, grbReviewers }: GRBReviewProps) => {
             />
           </Alert>
 
+          {/* Supporting Docs text */}
+          <h2 className="margin-bottom-0">{t('supportingDocuments')}</h2>
+          <p className="margin-top-05 line-height-body-5">
+            {t('supportingDocumentsText')}
+          </p>
+
+          {/* Business Case Card */}
+          <div className="usa-card__container margin-left-0">
+            <CardHeader>
+              <h3 className="display-inline-block margin-right-2">
+                {t('businessCaseOverview.title')}
+              </h3>
+              <span className="text-base">
+                {t('businessCaseOverview.submitted')}{' '}
+                {formatDateLocal(businessCase.updatedAt, 'MM/dd/yyyy')}
+              </span>
+            </CardHeader>
+            <CardBody>
+              <h4>{t('businessCaseOverview.need')}</h4>
+              <p>{businessCase.businessNeed}</p>
+              <h4>{t('businessCaseOverview.preferredSolution')}</h4>
+              <p>{businessCase.preferredSolution.summary}</p>
+            </CardBody>
+            <CardFooter>
+              <UswdsReactLink
+                to="./business-case"
+                className="display-flex flex-row flex-align-center"
+              >
+                <span className="margin-right-1">
+                  {t('businessCaseOverview.linkToBusinessCase')}
+                </span>
+                <IconArrowForward />
+              </UswdsReactLink>
+            </CardFooter>
+          </div>
+
+          {/* Additional Documents Title and Link */}
+          <h3 className="margin-bottom-1">{t('additionalDocuments')}</h3>
+          <UswdsReactLink
+            to="./documents/upload"
+            className="display-flex flex-align-center"
+          >
+            <IconAdd className="margin-right-1" />
+            <span>{t('additionalDocsLink')}</span>
+          </UswdsReactLink>
```

Comment:
This link should only show if the user is a GRT admin

Reply:
Added in [f2c84ba](https://github.com/CMS-Enterprise/easi-app/pull/2776/commits/f2c84ba292c27c814f60340ec34482be643dd977)